src/qml/PositioningInformationView.qml

```diff
@@ -8,372 +8,59 @@ import Theme
 Rectangle {
   id: positioningInformationView
 
-  property Positioning positionSource
-  property var coordinates: GeometryUtils.reprojectPoint(positionSource.sourcePosition, CoordinateReferenceSystemUtils.wgs84Crs(), projectInfo.coordinateDisplayCrs)
-  property bool coordinatesIsXY: CoordinateReferenceSystemUtils.defaultCoordinateOrderForCrsIsXY(projectInfo.coordinateDisplayCrs)
-  property bool coordinatesIsGeographic: projectInfo.coordinateDisplayCrs.isGeographic
+  property alias positionSource: positioningInformationModel.positioningSource
+  property alias antennaHeight: positioningInformationModel.antennaHeight
 
-  property double distanceUnitFactor: UnitTypes.fromUnitToUnitFactor(Qgis.DistanceUnit.Meters, projectInfo.distanceUnits)
-  property string distanceUnitAbbreviation: UnitTypes.toAbbreviatedString(projectInfo.distanceUnits)
-
-  property double antennaHeight: NaN
-  property double cellHeight: 26
-  property double cellPadding: 6
   property color backgroundColor: "transparent"
   property color alternateBackgroundColor: Theme.positionBackgroundColor
   property color textColor: positionSource.currentness ? Theme.mainTextColor : Theme.secondaryTextColor
-
-  property real contentHeight: grid.rows * positioningInformationView.cellHeight
-  width: parent.width
-  anchors.margins: 20
+  property double cellHeight: 26
+  property double cellPadding: 6
+  property real contentHeight: grid.count / grid.numberOfColumns * cellHeight
 
   color: Theme.mainBackgroundColorSemiOpaque
+  anchors.margins: 20
+  width: parent.width
 
-  Grid {
+  GridView {
```

Comment:
Sweet -- did you check that the details ordering remain the same as what we had before, inc. for both 2 cells per row vs. 4 cells per row?

Reply:
yes.